src/stylesheets/components/_search.scss

```diff
@@ -11,9 +11,11 @@
   @include typeset($theme-search-font-family);
   position: relative;
 
-  // if role is in the <form> element (>=2.5.0)...
+  // if role is in the <form> element (>=2.6.0)...
   &[role="search"],
-  // ...or if the component has a separate <div> (<2.5.0)
+  // if there is a (=2.5.[0,1])...
+  &[role="search"] div,
```

Comment:
Should we be extra-cautious and do `&[role="search"] > div`?

Reply:
Yes

Reply:
I'll do that